inc/global_assets/namespace.php

```diff
@@ -108,25 +108,10 @@ function configure_site() {
 /**
  * Filter the WordPress media provider name.
  *
- * @param string $name The default provider name.
  * @return string
  */
-function set_global_site_provider_name( string $name ) : string {
-	static $provider_name;
-
-	// Cache the value so switch_to_blog() only happens once.
-	if ( ! empty( $provider_name ) ) {
-		return $provider_name;
-	}
-
-	$provider_name = $name;
-
-	$site_name = get_blog_option( Global_Content\get_site_id(), 'blogname' );
-	if ( ! empty( $site_name ) ) {
-		$provider_name = sprintf( '%s %s', $site_name, __( 'Media' ) );
-	}
-
-	return $provider_name;
+function set_global_site_provider_name() : string {
+	return __( 'Global Media Library', 'altis' );
```

Comment:
With the reduced scope of this function, it feels like this value should be filterable if people want to change the name.

Reply:
It can be filtered upstream in the AMF WP plugin - that's what this is hooking into :)

Reply:
Okay, maybe I'm remembering that I saw that already. In that case the actual site name is irrelevant and this is good. 👍